src/library/scala/collection/StringOps.scala

```diff
@@ -848,9 +848,13 @@ final class StringOps(private val s: String) extends AnyVal {
 
   /** You can follow a string with `.r`, turning it into a `Regex`. E.g.
     *
-    *  `"""A\w*""".r`   is the regular expression for identifiers starting with `A`.
+    *  `"""A\w*""".r`   is the regular expression for ASCII-only identifiers starting with `A`.
+    *
+    *  `"""(?<month>\d\d)-(?<day>\d\d)-(?<year>\d\d\d\d)""".r` matches dates
```

Comment:
month-day-year, srsly?? :table_flip:

Reply:
I copied the example from the now-deprecated `def r(groupNames)` below.

Reply:
I noticed the comment compaction, but my thought was not to give examples but link to the copious doc on Regex, which correctly has year first :) but doesn't have an inline group names example.

Reply:
> month-day-year

🇺🇸 🎆